manifests/server/install.pp

```diff
@@ -1,5 +1,7 @@
 # == Class dns::server
 #
+include dns::server::default
```

Comment:
It does, but you shouldn't. All resources need to be contained in some sort of class, not just setting out there for the parser to suck up as it reads the file.